lib/utils.js

```
@@ -11,6 +11,22 @@ const Utils = {
     }
     return ''
   },
+  handleError(component) {
+    return error => {
+      if (error.response.status !== 422) {
+        return
+      }
+
+      let errors = []
+      let enrollmentError
+      for (enrollmentError in error.response.data) {
+        if (Object.prototype.hasOwnProperty.call(error.response.data, enrollmentError)) {
+          errors = errors.concat(error.response.data[enrollmentError])
+        }
+      }
+      component.setState({errors})
```

This function does not actually handle error as it silently fails on error other than 422. We should handle unexpected error also.